template-html5-app/src/brightsign/device-mock/raptor.ts

```diff
@@ -0,0 +1,9 @@
+const info = {
```

Comment:
I prefer in-line export - export const info. Gets rid of this error/handles that stuff for you